src/command_modules/azure-cli-appservice/setup.py

```diff
@@ -37,7 +37,7 @@
     'urllib3[secure]>=1.18',
     'xmltodict',
     'pyOpenSSL',
-    'vsts-cd-manager',
+    'vsts-cd-manager<1.0',
```

Comment:
Since `vsts-cd-manager` is not used at anywhere else, so this should be fine as pip will pick the highest; but if in future some other components lock to a lower version then you will get that one, rather the highest.